bin/BuildPackages.sh

```diff
@@ -236,17 +209,13 @@ build_one_package() {
       build_carat
     ;;
 
-    cohomolo*)
```

Comment:
Is the ````cohomolo```` case not needed anymore? I have no idea, not using it, just (double) checking....